src/dcqc/utils.py

```diff
@@ -11,7 +11,12 @@ def open_parent_fs(url: str) -> tuple[FS, str]:
         prefix = scheme + separator
 
     # parent_path can be "" if there is no "/" in the path
-    parent_path, _, base_name = path.rpartition("/")
+    parent_path, _, base_name = path.partition("/")
```

Comment:
```
path = "my/path/to/parent/basename"
path.partition("/")
# ('my', '/', 'path/to/parent/basename')
```

This is being pedantic, but my brain is so used to `base_name` being `basename`.  Can you speak to the update here? (No need to update the variable names - I'm just curious)

Reply:
I added a comment and updated the variable names to clarify this change. For example, `basename` is better called `path` to use the FS terminology. 

```
    # Retrieve the "top-most" parent folder for the FS root
    # The remainder of the string can be used as the FS path
    fs_root, _, path = resource.partition("/")
```